### PR TITLE
Illumina seq HW

### PR DESCRIPTION
**Pull request by rbortner**

### CODE REVIEW

File: assignments/002-unix-intro/unix-intro.sh
Comment (eriqande) [inline assignments/002-unix-intro/unix-intro.sh:1]: It looks like you might have pulled main into unix-intro and then branched off of that to make illumina-seq.  Try to switch branches back to main, syncing and pulling that, and then branching off of main.  That way, all there is in this branch would be the new stuff (i.e. the illumina seq stuff).

File: assignments/003-illumina-sequencing-questions/illumina-seq-homework.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+DNA is synthesized in the 5' - 3' direction. Therefore, we conclude that the Template must be anchored by it's 3' end (because the template would be read 3' -5'). 
+
+* Q5: How does the remaining strand relate to the Original Template? Is the remaining strand anchored by its 3' or 5' end? (_from 85.0 to 88.0 seconds in the video_).
+
+The remaining strand is the sequence complement to the template strand, with the matching A-T and C-G bases depending on the template sequence. It runs the opposite direction to the template, and so is anchored by it's 5' end. 
+
+* Q6: Choose one from the parentheses: the purple part on the remaining strand is (the same as || complementary to) the purple lawn oligos.  (_from 88.2 to 92.2 seconds in the video_).
+
+Complementary
```
Comment (eriqande) [inline assignments/003-illumina-sequencing-questions/illumina-seq-homework.md:38]: No. It's the same.  In fact, it started off as a lawn oligo itself.

**Comment by eriqande** [comment]
Great work Robyn.  I get a different answer for question 6.  
See key at:
https://eriqande.github.io/erics-captioned-vids/vids/illumina-sbs/

Look it over and see if I am correct or not on that!  Lots of folks have differed from me on 6!

**Comment by rbortner** [comment]
Hi Eric,

So, I chose 'complimentary' because at 1:11 they say 'This oligo is complimentary to the adapter region on one of the fragment strands'. HOWEVER now that you brought that up, and I watched it again for the 138th time (lol), I see what you are asking. The purple part left on this strand IS the oligo that was on the lawn in the first place! The previous complement was washed away. I understand now 🙂 You really have to look for the little stems on the oligos in the video to keep track of them!
________________________________

**Comment by eriqande** [comment]
Right on!  Yes, it takes about 130 to 140 viewings---so true!

**Comment by rbortner** [comment]
Well, I have no idea what I did lol, but I will try again!
________________________________

File: assignments/002-unix-intro/unix-intro.sh
Comment (rbortner) [reply assignments/002-unix-intro/unix-intro.sh:1]: OK no clue what I did the first time, but I will try again :-)

**Comment by eriqande** [comment]
No worries, about doing it for this one, but you can just make sure you are
on main when you make the next branch for the next homework!

eric

**Comment by rbortner** [comment]
I'm feeling keen to try to get it right 🙂 I think I found the issue—unix hw was in my main branch so got cloned over to the illumina branch. Think it's sorted now—take a look if you get a chance!
________________________________

**Pull request closed without merging**